tools/functional-tester/etcd-tester/key_stresser.go

```diff
@@ -46,9 +47,8 @@ type keyStresser struct {
 
 	cancel func()
 	conn   *grpc.ClientConn
-
-	success int
-	failure int
+	// keyOps records the number of keys created and deleted during a test case
```

Comment:
s/during a test case/by the stresser.

the stresser doesn't know about test cases